s3/replication/manager/src/s3replicationmanager/app.py

```diff
@@ -17,11 +17,176 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-from s3replicationcommon.log import Logger
+# import sys
+# import argparse
+# import json
+from aiohttp import web
+import logging
+from urllib.parse import urlparse, parse_qs
+from .config import Config
+from s3replicationcommon.log import setup_logging
+
+LOG = logging.getLogger('manager_proc')
+
+HOST = '127.0.0.1'
+PORT = 8080
+
+# Dictionary holding job_id and fdmi record
+#  e.g. jobs = {'jobA': {'K1': 'V1'}}
+jobs = {}
+
+# List of active subscriber
+subscribers = []
+
+# List of jobs in progress
+jobs_inprogress = {}
+
+# Route table declaration
+routes = web.RouteTableDef()
+
+@routes.get('/jobs')
+async def list_jobs(request):
+    """List active jobs
+
+    Handler api to handle multiple query
+
+    """
+    url = request.path_qs
+    url_ob = urlparse(url)
+    query = parse_qs(url_ob.query)
+
+    # Return in progress jobs
+    if 'inProgress' in query:
+        LOG.debug('InProgress query param: {}'.format(query['inProgress']))
+        return web.json_response({'inProgress': jobs_inprogress})
+    # Return job counts
+    elif 'count' in query:
+        return web.json_response({'Number of jobs present': len(jobs)})
+    # Return requested jobs
+    elif 'prefetch' in query:
+        req_list1 = query['prefetch']
+        pf_cnt = req_list1[0]
+        req_list2 = query['subscriber_id']
+        sub_id = req_list2[0]
+        LOG.debug('sub_id is : {}'.format(sub_id))
+        LOG.debug('subscribers are : {}'.format(subscribers))
+
+        if sub_id in subscribers:
+            LOG.debug('jobs keys : {}'.format(list(jobs.keys())))
```

Comment:
May be just print the count, printing all keys will be too verbose.

Reply:
done